### PR TITLE
ERE official language

### PR DESCRIPTION
**Pull request by loup99**
- Added a decision with necessary description, tooltip and potential conditions to adopt the Greek language as the ERE

### CODE REVIEW

File: WTWSMS/common/event_modifiers/WtWSMS_event_modifiers.txt [hunk 1]
```diff
@@ -210,27 +210,8 @@ persecuting_nestorians = { icon = 15 }
...
-	global_revolt_risk = 0.15
-	city_vassal_tax_modifier = -0.1
-	temple_vassal_tax_modifier = 0.05
-	culture_flex = -0.2
-}
-greek_official_language_2 = {
-	global_revolt_risk = -0.05
-	city_vassal_tax_modifier = 0.05
-	temple_vassal_tax_modifier = -0.1
+	city_tax_modifier = 0.05
```
Comment (IhateTrains) [inline WTWSMS/common/event_modifiers/WtWSMS_event_modifiers.txt]: What do you think about adding some malus to counterbalance?
e.g. monthly_character_prestige = -0.05

Comment (loup99) [reply WTWSMS/common/event_modifiers/WtWSMS_event_modifiers.txt]: The malus is the one time hit at the moment.

Comment (IhateTrains) [reply WTWSMS/common/event_modifiers/WtWSMS_event_modifiers.txt]: I like that it requires the themata system, but does it exist? I have searched the mod files and I can't find other occurences of `themata_system` flag. Is it planned?

Comment (loup99) [reply WTWSMS/common/event_modifiers/WtWSMS_event_modifiers.txt]: Planned.

**Comment by IhateTrains** [comment]
I couldn't test the decision in game because the game crashes on this branch, but looking at the code I seen no problems 👍

**Pull request merged**